tests/ludwig/modules/test_reduction_modules.py

```diff
@@ -0,0 +1,24 @@
+import logging
+from typing import Tuple
+
+import pytest
+import torch
+
+from ludwig.modules import reduction_modules
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.parametrize("reduce_mode", ["last", "sum", "mean", "avg", "max", "concat", None])
+@pytest.mark.parametrize("test_input_shape", [(16, 1, 4), (4, 10, 16)])
+def test_sequence_reducer(reduce_mode: str, test_input_shape: Tuple[int, ...]):
+    batch_size = test_input_shape[0]
+    max_sequence_length = test_input_shape[1]
+    sequence_reducer = reduction_modules.SequenceReducer(reduce_mode=reduce_mode)
+    inputs = torch.zeros(test_input_shape)
```

Comment:
Can you make sure these tests are GPU compliant? Involves moving the input tensor and module to GPU if available.

E.g. [here](https://github.com/ludwig-ai/ludwig/blob/1daab8714463e63c8f4b4ba4caed3cc1ef97a195/tests/ludwig/encoders/test_h3_encoders.py)